src/utils/components/LazyActionMenu/overrides.ts

```
@@ -95,18 +95,15 @@ export const mergeOptions = (options: MenuOption[]): MenuOption[] =>
-      (acc, opt) => {
-        const existingIndex = acc.findIndex(({ id }) => id === opt.id);
-        if (existingIndex === -1) {
-          acc.push({ duplicates: [opt], id: opt.id });
-        } else {
-          acc[existingIndex].duplicates.push(opt);
-        }
-        return acc;
-      },
-      [] as { duplicates: MenuOption[]; id: string }[],
-    )
+    .reduce((acc, opt) => {
```

ESLint

[error] Insert `⏎······`

(prettier/prettier)